spark/v3.3/build.gradle

```diff
@@ -140,6 +140,9 @@ project(":iceberg-spark:iceberg-spark-extensions-${sparkMajorVersion}_${scalaVer
       exclude group: 'org.roaringbitmap'
     }
 
+    testImplementation project(':iceberg-parquet')
+    testImplementation "org.apache.parquet:parquet-column"
```

Comment:
We don't have to put the test in this module. Basically, a parquet file with multiple row groups will trigger the bug in class `TestSparkReaderDeletes`.

Reply:
Let me look into your test pointers.
I put my repro in spark-extensions because I was simply following the example given in https://github.com/apache/iceberg/issues/5927. And in order to use UPDATE, we need the spark-extensions JAR to be present.

Reply:
Added a test in `TestSparkReaderDeletes` as you suggested, so there is no need to modify the spark-extensions build.